### ISSUE TITLE
Sending to multiple devices

### ISSUE THREAD

**Issue by Titaniumtown** [seemoo-lab/opendrop#29]
I was wondering if there was a plan for sending to multiple devices simultaneously? Or even better, having a argument to either `opendrop find` or `opendrop send` that would find a device then automaticly attempt to send a file to it.

Possible example of sending a file to multiple devices:
`opendrop send -f "file.png" -n "device name" -r 0,1,2,3,4`

Possible example of automaticly finding and sending file:
`opendrop find --auto-send -n "device name" -f "file.png"`

**Comment by schmittner** [comment]
Regarding your first idea: you could easily script this.

Regarding your second idea: I thought about such a design but decided against it since you should know who you are sending the file to before actually sending the file. The indices are not deterministic as they depend on the order in which devices are discovered. I know the current solution is not the most elegant but at least gives you control over who the receiver will be.

**Comment by Titaniumtown** [comment]
Regarding the second idea, opendrop find should automaticly save the results, so you don't have to constantly stop and start it.

**Comment by schmittner** [comment]
The results *are* stored on disk. You don't have to run `find` every time.

**Comment by Titaniumtown** [comment]
Yes I know that but to update the discover.last.json file, you have to press enter to stop opendrop find. What I am trying to say is that opendrop find should automatically update the discover.last.json file without the need of pressing enter, which stops opendrop find.
